src/components/menu-bar/menu-bar.css

```diff
@@ -56,11 +57,11 @@
     line-height: $menu-bar-height; 
     cursor: pointer;
     text-decoration: none;
-    color: white;
+    color: $motion-tertiary;
     user-select: none;
 }
 
-.menu-item:hover {
+/*.menu-item:hover {
```

Comment:
Did we want to remove this completely instead of commenting out?

Reply:
@paulkaplan I'm fine with that – I thought I'd leave it in for when we re-enable, but you're right, we can always just re-implement rather than uncomment!